lib/model/CModelDetailsView.cc

```diff
@@ -16,6 +16,7 @@
 #include <model/CEventRatePopulationModel.h>
 #include <model/CMetricModel.h>
 #include <model/CMetricPopulationModel.h>
+#include <model/CSearchKey.h>
```

Comment:
Did this need to be added?

Reply:
No this was accidental while testing with some debug. Removed in [a1742db](https://github.com/elastic/ml-cpp/pull/1551/commits/a1742db52970e90876639780cf33f8d7a14c7d80).